src/app/notes/notes.component.html

```diff
@@ -0,0 +1,16 @@
+<div class="flex">
+  <app-note-list
+    class="list-component"
+    [noteList]="noteList"
+    [displayedColumns]="displayedColumns"
+    (addNote)="noteService.addNote()"
+    (editNote)="noteService.setCurrentNote($event)"
+    (deleteNote)="deleteNote($event)"
+  ></app-note-list>
+  <app-note-details *ngIf="noteService.currentNote"
+    class="detail-component"
+    [note]="noteService.currentNote"
```

Comment:
Does this need to come from the note service? We have a currentNote property in the notes component we should be able to use